test/unittest/tests/block/test_block_session_size.cpp

```diff
@@ -0,0 +1,286 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+/*
+ * [block_session_size]: block_session.c
+ * The block manager extent list consists of both extent blocks and size blocks. This unit test
+ * suite tests aims to test all of the allocation and frees of the size block functions.
+ *
+ * The block session manages an internal caching mechanism for both block and size blocks that are
+ * created or discarded.
+ */
+#include "wt_internal.h"
+#include <catch2/catch.hpp>
+#include "../wrappers/mock_session.h"
+
+void
```

Comment:
It seems like these functions are mostly used within this file only. Do we have guidelines on whether we should always be using `static`?

Reply:
Where the functions are only used in this file, making them static makes sense. If we later need them elsewhere, we can move them then.

Reply:
I will be moving these functions in a new ticket very shortly.